src/css/index.scss

```diff
@@ -18,6 +18,7 @@
 @import '~@mozilla-protocol/core/protocol/css/components/menu-item';
 @import '~@mozilla-protocol/core/protocol/css/components/modal';
 @import '~@mozilla-protocol/core/protocol/css/components/navigation';
+@import '~@mozilla-protocol/core/protocol/css/components/notification-bar';
```

Comment:
Existing component in the design system. See https://protocol.mozilla.org/patterns/molecules/notification-bar.html#